packages/fixie-sdk/src/request-wrapper.tsx

```diff
@@ -49,6 +65,7 @@ export function FixieRequestWrapper({
               <Json>
                 {{
                   kind: 'text',
+                  state: <MessageState>{message.element}</MessageState>,
```

Comment:
Q: is there no double rendering because the element is memoized by default? Is that only true of OpenAI or all models/mechanisms?

Reply:
Excellent question :)

The (not at all obvious) answer is that elements surfaced by partial rendering are automatically memoized (this was not always the case), so any time we see a "conversational" element, it's already been memoized.